src/Build/Evaluation/Expander.cs

```diff
@@ -3933,6 +3987,28 @@ private static bool TryGetArgs(object[] args, out int arg0, out string arg1)
                 return false;
             }
 
+            private static bool TryGetArgs(object[] args, out string arg0, out int arg1)
+            {
+                arg0 = null;
+                arg1 = 0;
+
+                if (args.Length != 2)
+                {
+                    return false;
+                }
+
+                var value1 = args[1] as string;
+                arg0 = args[1] as string;
```

Comment:
Something about these indexes seems confused. Should it be 
```suggestion
                arg0 = args[0] as string;
```

?